package.json

```diff
@@ -7,7 +7,7 @@
     "src"
   ],
   "engines": {
-    "node": ">= 6"
+    "node": ">= 10.13"
```

Comment:
are you sure? @evilebottnawi said the supported version was 10.13

Reply:
@michael-ciniawsky webpack@5 works only with `10.13`, so no one will use a loader on `10.12`/`10.11`/etc, webpack@4 most of plugins and loaders already require `10.13`, so again no one will use a loader on `10.12`/`10.11`/etc

Reply:
If there is a **reason** (e.g a known bug) for `webpack` to specify a minium on `>= 10.13` the loader should of course follow suit here. But in case the version was simply updated to `>= 10.13` because it was the lastest minor in the current `v10.x` LTS line at that moment I'm not willing to deal with any particular issue(s) reported that user x gets an annoying warning because he/she is still on `=< 10.13` but it "works" for them. Specific versions ranges are fine to use when there is a reason, but should be avoided otherwise...

Reply:
@michael-ciniawsky many transitive deps require `10.13`, because node recommend to support only LTS version, so i think developers just even can't run webpack on `10.12` and less